schema.sql

```diff
@@ -70,8 +70,17 @@ CREATE TABLE specializations (
 );
 
 CREATE TABLE visits (
-      animal_id INT NOT NULL   REFERENCES animals(id),
+      animal_id INT NOT NULL REFERENCES animals(id),
       vet_id INT NOT NULL  REFERENCES vets(id),
       visit_date DATE NOT NULL   
 );
 
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Create indexes
+CREATE INDEX animal_ids ON visits (animal_id);
+
+CREATE INDEX vet_ids ON visits (vet_id);
+
+CREATE INDEX owner_emails ON owners (email);
```

Comment:
Execution time for both of these queries should be less than 1000 ms. Kindly rerun the queries and see if execution time decreases.

Reply:
Done ✅ 
<img width="1099" alt="3" src="https://user-images.githubusercontent.com/53530780/191302183-3b000881-71cb-4a3e-9af6-5e7555bd88b0.png">